.gitignore

```diff
@@ -296,3 +292,10 @@ pylint-output.txt
 
 # Ignore mock test files
 /code-monkey/src/tests/mock_files/
+
+# Ignore the ruff cache
+.ruff_cache/
+
+# Our own stuff
+artifacts/
+.prompt.md
```

Comment:
had to move `artifacts/` to the end because there are other patterns above where we specify that we want to keep some files (`.env.secrets.sample` might be the other one), and we end up copying it over and over into artifacts/